docs/commitStyle.md

```diff
@@ -0,0 +1,55 @@
+## Commit Style Guidelines for Susi Android
```

Comment:
For `Susi Android`? Change it to Badge-Magic Android.

Reply:
XD

Reply:
😅